bot/exts/fun/uwu.py

```diff
@@ -0,0 +1,132 @@
+
+
+class Uwu(Cog):
+    """Cog for the uwu command."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    def _word_replace(self, input_string: str) -> str:
+        """Replaces words that are keys in the word replacement hash to the values specified."""
+        for word, replacement in WORD_REPLACE.items():
+            input_string = input_string.replace(word, replacement)
+        return input_string
+
+    def _char_replace(self, input_string: str) -> str:
+        """Replace certain characters with 'w'."""
+        return REGEX_WORD_REPLACE.sub("w", input_string)
+
+    def _stutter(self, strength: float, input_string: str) -> str:
+        """Adds stuttering to a string."""
+        return REGEX_STUTTER.sub(partial(self._stutter_replace, strength=strength), input_string, 0)
+
+    def _stutter_replace(self, match: re.Match, strength: float = 0.0) -> str:
+        """Replaces a single character with a stuttered character."""
+        match_string = match.group()
+        if random.random() < strength:
+            return f"{match_string}-{match_string[-1]}"  # Stutter the last character
+        return match_string
+
+    def _nyaify(self, input_string: str) -> str:
+        """Nyaifies a string."""
+        return REGEX_NYA.sub(SUBSTITUTE_NYA, input_string, 0)
+
+    def _emoji(self, strength: float, input_string: str) -> str:
+        """Replaces some punctuation with emoticons."""
+        return REGEX_PUNCTUATION.sub(partial(self._emoji_replace, strength=strength), input_string, 0)
+
+    def _emoji_replace(self, match: re.Match, strength: float = 0.0) -> str:
+        """Replaces a punctuation character with an emoticon."""
+        match_string = match.group()
+        if random.random() < strength:
+            return f" {random.choice(EMOJIS)} "
+        return match_string
+
+    def _uwuify(self, input_string: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1) -> str:
+        """Takes a string and returns an uwuified version of it."""
+        input_string = input_string.lower()
+        input_string = self._word_replace(input_string)
+        input_string = self._nyaify(input_string)
+        input_string = self._char_replace(input_string)
+        input_string = self._stutter(stutter_strength, input_string)
+        input_string = self._emoji(emoji_strength, input_string)
+        return input_string
+
+    @commands.command(name="uwu", aliases=("uwuwize", "uwuify",))
+    async def uwu_command(self, ctx: Context, *, text: clean_content(fix_channel_mentions=True)) -> None:
+        """
+        Echo an uwuified version the passed text.
+
+        Example:
+        '.uwu Hello, my name is John' returns something like
+        'hewwo, m-my name is j-john nyaa~'.
+        """
+        text, embed = await Fun._get_text_and_embed(ctx, text)
+
+        # Grabs the text from the embed for uwuification.
+        if embed is not None:
+            embed = Fun._convert_embed(self._uwuify, embed)
```

Comment:
We should change this to use `get_cog()`.
 ```suggestion
         if (fun_cog := ctx.bot.get_cog("Fun")):
             text, embed = await fun_cog._get_text_and_embed(ctx, text)

            # Grabs the text from the embed for uwuification.
            if embed is not None:
                embed = fun_cog._convert_embed(self._uwuify, embed)
        else:
             embed = None
```

Reply:
Linting fails for the suggestion